src/data/graphql/queries.graphql

```diff
@@ -637,3 +637,23 @@ query SubgraphSingleDomain($colonyAddress: String!, $domainId: Int!) {
     }
   }
 }
+
+query SubgraphEvents($colonyAddress: String!) {
+  events(where: { associatedColony: $colonyAddress }) {
+    id
+    address
+    associatedColony {
+      colonyAddress: id
+      id: colonyChainId
+    }
+    transaction {
+      hash: id
+      block {
+        id
```

Comment:
i dont think we need to query it

Reply:
It doesn't hurt either

Reply:
I remember you was ask me to remove it in my PR, so thought that you actually dont like it 😏

Reply:
Haha, I honestly don't remember that.

But fine. I'll remove it then :)